app/Config.hs

```diff
@@ -16,10 +17,8 @@ import System.Hapistrano.Commands
 data Config = Config
   { configDeployPath :: !(Path Abs Dir)
     -- ^ Top-level deploy directory on target machine
-  , configHost :: !(Maybe String)
-    -- ^ Host to deploy to. If missing, localhost will be assumed.
-  , configPort :: !Word
-    -- ^ SSH port number to use, may be omitted
+  , configHosts :: ![(String, Word)]
```

Comment:
@mrkkrp yeah I know `Map` is an external dependency from `containers`. So if we could just use what it is on `base` would be better for now.